src/main/java/com/marklogic/appdeployer/command/PayloadPropertyFilter.java

```diff
@@ -0,0 +1,55 @@
+package com.marklogic.appdeployer.command;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.mgmt.PayloadParser;
+import com.marklogic.rest.util.Fragment;
+import org.jdom2.Element;
+import org.jdom2.output.XMLOutputter;
+
+public class PayloadPropertyFilter {
+
+	private PayloadParser payloadParser = new PayloadParser();
+
+	public String includeProperty(String payload, String propertyName, String propertyValue) {
```

Comment:
I think "include" should be a way of specifying only those properties to include. Which means everything that is not one of those properties should be removed. 

And similar to below, how about "includeProperties(String payload, String... propertyNames)"?

Reply:
Got it. Completely misunderstood what you meant